resources/istio/README.md

```diff
@@ -4,21 +4,17 @@
 
 [Istio](https://istio.io/) is an open-source service mesh providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies, and aggregate telemetry data.
 
-The documentation here is for developers only, please follow the installation instructions from [istio.io](https://istio.io/docs/setup/install/istioctl/) for all other use cases.
+The Istio Helm chart consists of the `istio-manager-config.yaml` file with configuration needed for installation of Istio module.
 
-The Istio Helm chart consists of the `istio-operator.yaml` file with tailored Istio including Kyma-specific changes and configuration options.
-
-By default, this chart installs the following Istio components:
+Istio module by default installs the following Istio components:
 
 - ingressgateway
-- egressgateway
 - istiod (pilot, citadel, and galley)
-
-To disable any of the components, change the corresponding `enabled` flag.
+- istio-cni
 
 ## Installation
 
-The installation of the Istio chart requires [Reconciler](https://github.com/kyma-incubator/reconciler/tree/main/pkg/reconciler/instances/istio). Reconciler uses `istioctl` and a rendered `istio-operator.yaml` file to install Istio on a cluster. To install the component, run:
+The installation of the Istio chart requires [Reconciler](https://github.com/kyma-incubator/reconciler/tree/main/pkg/reconciler/instances/istio). Reconciler uses [Istio library] and `IstioOperatorConfiguration` file to install Istio on a cluster. To install the component, run:
```

Comment:
```suggestion
The installation of the Istio chart requires [Reconciler](https://github.com/kyma-incubator/reconciler/tree/main/pkg/reconciler/instances/istio). Reconciler uses the [Istio library](https://github.com/istio/istio/tree/master/operator) and the `IstioOperatorConfiguration` file to install Istio on a cluster. To install the component, run:
```
"Istio library" is in brackets but the link was missing. Did you want to add this one?